policies/releasestrat.html

```diff
@@ -20,27 +20,61 @@ <h5>
-	  fixes and no new features. Minor releases that change the
-	  last digit, e.g. 1.1.0 vs. 1.1.1, can and are likely to
-	  contain new features, but in a way that does not break binary
-	  compatibility. This means that an application compiled and
-	  dynamically linked with 1.1.0 does not need to be recompiled
-	  when the shared library is updated to 1.1.1. It should be
-	  noted that some features are transparent to the application
-	  such as the maximum negotiated TLS version and cipher suites,
-	  performance improvements and so on. There is no need to
-	  recompile applications to benefit from these features.</p>
+	    As of release 3.0.0, the OpenSSL versioning scheme is changing
+	    to a more contemporary format: MAJOR.MINOR.PATCH
+	  </p>
+
+	  <p>
+	    With this format, API/ABI compatibility will be guaranteed
+	    for the same MAJOR version number. Previously we guaranteed
+	    API/ABI compatibility across the same MAJOR.MINOR combination.
+	  </p>
+
+	  <ul>
+	    <li>MAJOR: API/ABI incompatible changes will increase this number</li>
+	    <li>MINOR: API/ABI compatible feature releases will change this</li>
+	    <li>PATCH: Bug fix releases will increment this number. We also
+            allow backporting of accessor functions in these releases.</li>
+	  </ul>
+
+	  <p>
+	    This more closely aligns with the expectations of users who are
+	    familiar with semantic versioning.  However, we have not adopted
+	    semantic versioning in the strict sense of its rules, because it
+	    would mean changing our current LTS policies and practices.
+	  </p>
+
+	  <p>
+	    The current 1.1.1 and 1.0.2 versioning scheme remains unchanged:
+
+	    <blockquote><i>
+	      As of release 1.0.0 the OpenSSL versioning scheme was improved
+	      to better meet developers' and vendors' expectations. Letter
+	      releases, such as 1.0.2a, exclusively contain bug and security
+	      fixes and no new features. Releases that change the last digit,
+	      e.g. 1.1.0 vs. 1.1.1, can and are likely to
+	      contain new features, but in a way that does not break binary
+	      compatibility. This means that an application compiled and
+	      dynamically linked with 1.1.0 does not need to be recompiled
+	      when the shared library is updated to 1.1.1. It should be
+	      noted that some features are transparent to the application
+	      such as the maximum negotiated TLS version and cipher suites,
+	      performance improvements and so on. There is no need to
+	      recompile applications to benefit from these features.
+	    </i></blockquote>
+	  </p>
+
+	  <hr />
 
 	  <p>With regards to current and future releases the OpenSSL
 	  project has adopted the following policy:</p>
 
 	  <ul>
-	    <li>Version 1.1.0 will be supported until one year after the release of 1.1.1</li>
+	    <li>The next version of OpenSSL will be 3.0.0.</li>
+	    <li>Version 1.1.1 will be supported until 2023-09-11 (LTS).</li>
```

Comment:
Because it's an LTS release, I would have thought it would be supported until the end of the terminating year, is that not the case?

Reply:
No. 1.1.1 is supported for 5 years from its release date.